lbry/conf.py

```diff
@@ -461,6 +461,11 @@ def set_persisted(self, config_file_path=None):
         if self.persisted.upgrade():
             self.persisted.save()
 
+    @property
+    def is_running_from_bundle(self):
+        # see https://pyinstaller.readthedocs.io/en/stable/runtime-information.html
+        return getattr(sys, 'frozen', False) and hasattr(sys, '_MEIPASS')
+
```

Comment:
This isn't a configurable setting, please move this to `lbry.utils`.

Reply:
Done. However, some further thoughts: in my mind conf could easily include both user configuration and platform configuration. I'm fine with them separate also. It seems that platform configuration is generally missing or unorganized the lbry-SDK. It should be there in OOP form, a platform service of sorts, easily testible, stubible, mockable, etc.